test/ITensorChainRules/test_chainrules.jl

```diff
@@ -352,6 +352,17 @@ Random.seed!(1234)
     args = (A, B)
     test_rrule(ZygoteRuleConfig(), f, args...; rrule_f=rrule_via_ad, check_inferred=false)
   end
+
+  @testset "issue 1294" begin
```

Comment:
@mtfishman I added this test but it fails after 3 tests in `test_rrule` with the error 
```
test_rrule: f on ITensor,ITensor: Error During Test at /Users/kpierce/.julia/packages/ChainRulesTestUtils/by7Dj/src/testers.jl:202
  Got exception outside of a @test
  Fluxes not all equal
  ```
  I am not too familiar with this `test_rrule` function is this something I should look into fixing or is there a way to modify the test to not run the one that is expected to fail for the electron case